### ISSUE TITLE
[Bug]: Print tool shows even if option is not enabled in LM Plugin

### ISSUE THREAD

**Issue by gioman** [3liz/lizmap-plugin#557]
### What is the bug?

LMWC

![image](https://github.com/3liz/lizmap-web-client/assets/1951107/8e028aed-c32e-4908-8732-57262be169a9)

LM Plugin

![image](https://github.com/3liz/lizmap-web-client/assets/1951107/1c6ba5f7-3181-454c-a4fc-4f268465ecc9)

### Steps to reproduce the issue

See images above.

### Versions, safeguards, check summary etc

LMWC 3.7.3 and QGIS Desktop/Server 3.28.15 LM Plugin 4.2.0

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### Operating system

Ubuntu 22.04

### Browsers

Firefox, Chrome

### Browsers version

Latest updates

### Relevant log output

```shell
Nothing relevant.
```

**Comment by Gustry** [comment]
> That "critical" log message is curious

indeed, but this code is dodgy, I was trying to use the QGIS plugin manager, do not worry about this log. Let's see next release what is the output, CF 1aa2fbe257eb36cbaf9961afe525e2ae54a5eb92

> @Gustry just 1. I'm using a clean QGIS profile with LM plugin which points to just one fresh LMWC 3.7.3 installation.

Ok I got it. It's because the signel `lwc_version_changed` wasn't triggered in your case.
I will see what I can do. In the meantime, I made a quick fix : 9bcfeb13b19eea3b29bc642d6bf41445e8d3865c

**Comment by Gustry** [comment]
Do you still replicate this issue @gioman ?
I made a few bugfix about this signal to be emitted when the server has changed.
